image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-generate-seed.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-image-based-upgrade-generate-seed-image"]
+= Generating a seed image for the image-based upgrade with the {lcao}
+include::_attributes/common-attributes.adoc[]
+:context: generate-seed
+
+toc::[]
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` custom resource (CR).
```

Comment:
Optimization of content by rewriting something similar to:
`The telco clusters on bare-metal hardware require firmware updates for security, new features, and {product-title} compatibility.`